tests/fixtures/apps/mip/case_metrics_deliverables.yaml

```diff
@@ -0,0 +1,110 @@
+---
```

Comment:
If possible, could you look at other mip fixtures that can be removed?

Reply:
Done